packages/utilities/src/overflowHandler/README.md

```diff
@@ -0,0 +1,51 @@
+# overflowHandler
+
+`overflowHandler` is a framework-agnostic utility for handling the truncation of
+a list of items.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```js
+import { createOverflowHandler } from '@carbon/utilities';
+
+createOverflowHandler({
+  container, // The document node that holds the items. HTMLElement
+  maxVisibleItems, // Maximum number of visible items. If provided, only this number of items will be shown. number | undefined
+  onChange: (visibleItems: HTMLElement[], hiddenItems: HTMLElement[]) => void, // Callback function invoked when the visible and hidden items change.
+  dimension, // The dimension to consider for overflow calculations. "width" | "height" | undefined
+});
+```
+
+The only required argument necessary to consume this utility is `container`.
```

Comment:
Maybe I'm missing something, this statement seems inaccurate as `onChange` is required per `OverflowHandlerOptions`:

https://github.com/carbon-design-system/carbon/blob/4331e5fec7ae539e5b4ac6dac1d0ca8faede8aa6/packages/utilities/src/overflowHandler/overflowHandler.ts#L145

Further, if I change the `onChange` property in a TypeScript file to `undefined`, I get the following type error:

```
Type 'undefined' is not assignable to type '(visibleItems: HTMLElement[], hiddenItems: HTMLElement[]) => void'.
```